tainiisanta.js

```diff
@@ -0,0 +1,105 @@
+// есть n человек, необходимо распределить: кто кому дарит подарки
+// Нельзя дарить подарок самому себе
+
+function getRandomInt(max) { // при max = 3, результат одно из: 0,1,2
+    return Math.floor(Math.random() * max); // округляем число вниз и возвращаем полученное случайное число
+}
+
+function getLength (massiv) { // в этой программе из массивов удалять элементы я не буду, вместо этого буду заменять их значения на 0, а эта функция будет возвращать длину массива без "нулей".
+    let a=0;
```

Comment:
create understandable variable's name. When i try read code. I must undertand it by words.